heartbeat/monitors/task.go

```diff
@@ -166,3 +133,44 @@ func (t *configuredJob) Stop() {
 		t.client.Close()
 	}
 }
+
+func runPublishJob(job jobs.Job, client beat.Client) []scheduler.TaskFunc {
+	event := &beat.Event{
+		Fields: common.MapStr{},
+	}
+	next, err := job(event)
+	hasContinuations := len(next) > 0
+
+	if err != nil {
+		logp.Err("Job %v failed with: ", err)
+	}
+
+	if event.Fields != nil && !eventext.IsEventCancelled(event) {
```

Comment:
Previously there was an `event != nil` check here. I assume because of :138 this cannot happen?

Reply:
That is correct. My IDE actually flagged that as redundant.

Reply:
`event` is being passed as pointer to `job` which could nil it, but that's something we wont ever do as far as I am concerned